cmd/openfga/main_test.go

```diff
@@ -45,16 +43,18 @@ func (s *serverHandle) GetHTTPAddress() string {
 
 // runOpenFGAContainerWithArgs spins up an openfga container with the default configuration
 // exposed for testing purposes. It is assumed that the openfga/dockertest image is available.
-// The caller must call the returned function after the container is no longer needed.
 // This function asserts that the container's exit code was 0.
-func runOpenFGAContainerWithArgs(t *testing.T, commandArgs []string) (OpenFGATester, func()) {
+func runOpenFGAContainerWithArgs(t *testing.T, commandArgs []string) OpenFGATester {
 	t.Helper()
 
 	dockerClient, err := client.NewClientWithOpts(
 		client.FromEnv,
 		client.WithAPIVersionNegotiation(),
 	)
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		dockerClient.Close()
+	})
```

Comment:
```suggestion
	t.Cleanup(dockerClient.Close)
```

Reply:
doesn't work because `close()` returns an error 😬

Reply:
Oh, then why aren't we checking for the error?